appveyor.yml

```diff
@@ -13,7 +13,7 @@ environment:
       CONDA_CHANNELS: "conda-forge astropy-ci-extras"
       CONDA_DEPENDENCIES: "scipy matplotlib mock pandas requests beautifulsoup4 sqlalchemy scikit-image pytest-mock wcsaxes suds-jurko glymur"
       PIP_DEPENDENCIES: "hypothesis sunpy-sphinx-theme pytest-astropy"
-      NUMPY_VERSION: "stable"
+      NUMPY_VERSION: "1.13"
```

Comment:
😆 Windows problems.

Reply:
Oh yeah, I did not want to powershell some variables. 

I should though.

Reply:
I wouldn't worry

Reply:
Well I was going to until you pointed it out.

Hurry up NumPy 1.14.1